mqtt-ds/src/main/java/org/apache/rocketmq/mqtt/ds/notify/NotifyManager.java

```diff
@@ -73,18 +72,20 @@ public class NotifyManager {
     private NettyRemotingClient remotingClient;
     private DefaultMQProducer defaultMQProducer;
 
-
-    @Resource
-    private ServiceConf serviceConf;
+    private final ServiceConf serviceConf;
 
     @Resource
     private MetaPersistManager metaPersistManager;
 
     @Resource
     private FirstTopicManager firstTopicManager;
 
-    @PostConstruct
-    public void init() throws MQClientException {
+    public NotifyManager(ServiceConf serviceConf) {
+        this.serviceConf = serviceConf;
```

Comment:
Other than unit tests, I don't see anywhere to call it. The instance of Notifymanger cancelates the injection of ServiceConf, when is this ServiceConf passed in?

Reply:
+1, i think the constructor func is not necessary. @yx9o you can use `Mock` and `FieldUtils.writeDeclaredField` in unit test, for example, refer to `TestServiceConfListener.java`

Reply:
@Autowired to inject ServiceConf in the constructor is ok